### PR TITLE
Changes for bumping thunder and nanoservices to latest

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: conf/layer.conf [hunk 1]
```diff
@@ -50,7 +50,7 @@ GPU_MEM_1024 = "396"
 LICENSE_FLAGS_WHITELIST += "commercial_faad2"
 
 # Enable early start of Thunder
-DISTRO_FEATURES_append = " compositor thunder"
+DISTRO_FEATURES_append = " compositor thunder securityagent pxcore-libnode"
```
Comment (wouterlucas) [inline conf/layer.conf]: Please remove pxcore-libnode from the distro feature append. This shouldn't be added by us.

File: recipes-wpe/wpeframework/wpeframework-plugins_git.bb [hunk 2]
```diff
@@ -49,22 +43,30 @@ PACKAGECONFIG ?= " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'opencdm',              'opencdmi', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'clearkey',             'opencdmi_ck', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'compositor',           'compositor', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'fwupgrade-lib',        'firmwarecontrol', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready',            'opencdmi_pr', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_nexus',      'opencdmi_prnx', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_nexus_svp',  'opencdmi_prnx_svp', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'playready_vg',         'opencdmi_vgrdm', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'pxcore-libnode',       'spark', '', d)} \
```
Comment (wouterlucas) [inline recipes-wpe/wpeframework/wpeframework-plugins_git.bb]: this sounds like misuse of distro features, lets not add a distro feature for such a granular object. Please remove.

File: recipes-wpe/wpeframework/wpeframework-plugins_git.bb [hunk 3]
```diff
@@ -49,22 +43,30 @@ PACKAGECONFIG ?= " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'opencdm',              'opencdmi', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'clearkey',             'opencdmi_ck', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'compositor',           'compositor', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'fwupgrade-lib',        'firmwarecontrol', '', d)} \
```
Comment (wouterlucas) [inline recipes-wpe/wpeframework/wpeframework-plugins_git.bb]: Same here, misuse of distro featuers, please remove.

File: conf/layer.conf (hunk 1 above)
Comment (wouterlucas) [reply conf/layer.conf]: Same for security agent, either enable it as part of the defaults or leave it out and let other layers include it in the package config. But I do not like using DISTRO features for individual components.

**Comment by HaseenaSainul** [comment]
@rdkcmf-jenkins, hi team, could you please disable securityagent from the meta-cmf-raspberrypi-new19th/recipes-wpe/wpeframework/wpeframework-plugins_git.bbappend:  systemcommands DHCPServer securityagent ioconnector firmwarecontrol. There is two modules related to security agent in wpeframework and wpeframeworkplugins, so these has to be selected based DISTRO_FEATURE=security.  
Since security agent has to be enabled only the system which works based on the securityagent requirement. So suggest to remove this from the above line

**Comment by wouterlucas** [comment]
Merging it in anyway, RDK build needs to be fixed.

**Pull request merged**